config/jobs/testing/testing-presubmits.yaml

```diff
@@ -31,16 +31,14 @@ presubmits:
     trigger: "(?m)^/test config,?(\\s+|$)"
     rerun_command: "/test config"
 
-  - name: pull-testing-verify-images
-    always_run: false
-    context: pull-testing-verify-images
-    run_if_changed: "^(images/|legacy/images/).*$"
+  - name: pull-testing-verify-bazel
+    always_run: true
+    context: pull-testing-verify-bazel
     max_concurrency: 4
     agent: kubernetes
     decorate: true
     labels:
       preset-service-account: "true"
-      preset-dind-enabled: "true"
```

Comment:
Are we not using dind? Not too sure if like that.

Could we use long running dind instance (through stateful set maybe?) for enable docker caching

Reply:
The diff has rendered weirdly - as part of this PR, we'll no longer have a presubmit that builds images *at all*.

The rule in question that you're commenting on, is running `bazel build //...` (which is something we didn't previously run).

We can follow up with additional presubmits that automate testing images on each PR 😄

Reply:
> Could we use long running dind instance (through stateful set maybe?) for enable docker caching

Possibly, but this is also going to be super difficult to get set up and for now I just want to be able to push new images easily 😄

Reply:
Yeah makes more sense now with being aware no presubmit :slightly_smiling_face: